content/es/service.md

```diff
@@ -0,0 +1,8 @@
+---
+Title: Servicio
+Status: Terminada
+Category: concepto
+Tags: ["solicitud", "fundamental", ""]
+---
+
+Tenga en cuenta que en TI, el servicio tiene múltiples significados. En esta definición, nos centraremos en la más tradicional: servicio como en microservicio. Se matiza cómo o incluso si los servicios difieren de los microservicios y diferentes personas pueden tener opiniones diferentes. Para una definición de alto nivel, los trataremos como iguales. Consulte la definición de [microservicios](/microservices/).
```

Comment:
Also, keep all the break lines as it is in the original English term (instead of having just one paragraph)

```suggestion
Tenga en cuenta que en TI, servicio tiene múltiples significados. En esta definición, nos centraremos en la más tradicional: servicio como en microservicio. Se matiza cómo o incluso si los servicios difieren de los microservicios y diferentes personas pueden tener opiniones diferentes. Para una definición de alto nivel, los trataremos como iguales. Consulte la definición de [Arquitectura de Microservicios](/es/microservices-architecture/).
```